### ISSUE TITLE
Bad ARIA in footer that should be removed

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeBMD2#518]
The `ul` element for "Other FreeUK Genealogy Projects" has been given `role="navigation"` which is a mistake — it is bad ARIA and should be removed.

The navigation role robs the `ul` of its natural properties as a list container. The knock-on effects are: an automatic validator complains that there is no list container; a S-R will not understand; a S-R user will be misinformed about what is going on.

There is a `nav` element wrapping the list container anyway. (There _might_ be an issue around using `nav` elements in the footer, but I need to find out more before we consider doing anything to improve those elements.)

**Comment by DeniseColbert** [comment]
@Vino-S will make this change

**Comment by Vino-S** [comment]
Ready for testing in test3 CEN

**Comment by AlOneill** [comment]
@Vino-S looks good in t3 CEN — thank you.

**Comment by PatReynolds** [comment]
Fixed on FreeREG and FreeCEN.

Needs deployment to FreeBMD2.

**Comment by DeniseColbert** [comment]
Joseph could do this once he's in our code

**Comment by AlOneill** [comment]
This has already been fixed by @jayto581 as part of another issue, so closing.
